helm-chart/renku-notebooks/templates/deployment.yaml

```diff
@@ -35,13 +35,17 @@ spec:
             - name: JUPYTERHUB_API_TOKEN
               value: {{ .Values.jupyterhub_api_token }}
             - name: JUPYTERHUB_API_URL
+            {{ if .Values.jupyterhub_api_url }}
+              value: {{ .Values.jupyterhub_api_url }}
+            {{ else }}
               value: {{ template "notebooks.http" . }}://{{ .Values.global.renku.domain }}{{ .Values.jupyterhub_base_url }}hub/api
+            {{ end }}
             - name: JUPYTERHUB_SERVICE_PREFIX
               value: {{ .Values.jupyterhub_base_url }}services/notebooks/
             - name: JUPYTERHUB_BASE_URL
               value: {{ .Values.jupyterhub_base_url }}
-            - name: GITLAB_REGISTRY_SECRET
-              value: {{ template "renku.fullname" . }}-registry
```

Comment:
Why remove this?

Reply:
you're right, it shouldn't be removed -- not sure how to access secrets from the top-level chart in a sub-chart...

Reply:
I don't think it needs to be modified

Reply:
but if it is deployed standalone, the secret doesn't exist.

Reply:
ah ok. It is created manually at the moment (TODO: doc :stuck_out_tongue_closed_eyes: ) so nothing new here

Reply:
yeah at the moment we have no way of accessing private repos on a gitlab instance that we do not control. This will require a) handling user secrets in the notebook app or b) somehow getting tokens with repo access from e.g. the gateway